generator/src/test/java/com/scottlogic/deg/generator/cucumber/ShorterThan.feature

```diff
@@ -220,42 +212,47 @@ Scenario: Running a 'shorterThan' request alongside a non-contradicting shorterT
        | "♀aaa" |
 
 @ignore
-Scenario: Running a 'shorterThan' request alongside a greaterThan constraint should fail with an error message
-       And foo is shorter than 10
-       And foo is greater than 1
-     Then I am presented with an error message
-       And no data is created
-
+Scenario: Running a 'shorterThan' request alongside a greaterThan constraint should be successful
+       And foo is shorter than 2
+       And foo is greater than 10
+      Then the following data should be included in what is generated:
+       | foo    |
+       | "a"    |
+       | null   |
 
 @ignore
-Scenario: Running a 'shorterThan' request alongside a greaterThanOrEqualTo constraint should fail with an error message
-       And foo is shorter than 10
+Scenario: Running a 'shorterThan' request alongside a greaterThanOrEqualTo constraint should be successful
+       And foo is shorter than 2
        And foo is greater than or equal to 3
-     Then I am presented with an error message
-       And no data is created
+     Then the following data should be included in what is generated:
+       | foo    |
+       | "a"    |
+       | null   |
 
 @ignore
-Scenario: Running a 'shorterThan' request alongside a lessThan constraint should fail with an error message
-       And foo is shorter than 10
+Scenario: Running a 'shorterThan' request alongside a lessThan constraint should be successful
+       And foo is shorter than 2
        And foo is less than 15
-     Then I am presented with an error message
-       And no data is created
-
+     Then the following data should be included in what is generated:
+       | foo    |
+       | "a"    |
+       | null   |
 
 @ignore
```

Comment:
Can we raise a bug on this or link to an existing?

Reply:
This is being addressed in ticket #501

Reply:
This is being addressed in ticket #501